pytorch_lightning/callbacks/finetuning.py

```diff
@@ -393,10 +373,12 @@ def on_fit_start(self, trainer, pl_module):
             return super().on_fit_start(trainer, pl_module)
         raise MisconfigurationException("The LightningModule should have a nn.Module `backbone` attribute")
 
-    def freeze_before_training(self, pl_module: "pl.LightningModule"):
+    def freeze_before_training(self, pl_module: "pl.LightningModule") -> None:
         self.freeze(pl_module.backbone)
 
-    def finetune_function(self, pl_module: "pl.LightningModule", epoch: int, optimizer: Optimizer, opt_idx: int):
+    def finetune_function(
+        self, pl_module: "pl.LightningModule", epoch: int, optimizer: Optimizer, opt_idx: int
```

Comment:
```suggestion
        self, pl_module: "pl.LightningModule", epoch: int, optimizer: Optimizer, optimizer_idx: int
```
we have optimizer_idx in all other public methods in lightning.